content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -156,5 +156,5 @@ Let us know what you think about the plugin! Provide feedback on our [discussion
 [1]: https://github.com/DataDog/datadog-for-intellij/discussions
 [3]: /getting_started/profiler/
 [4]: https://www.datadoghq.com/
-[5]: /account_management/api-app-keys/
-[6]: https://plugins.jetbrains.com/plugin/19495-datadog
\ No newline at end of file
+[5]: /getting_started/site/#overview
```

Comment:
It looks like this link is no longer used and can be removed.

Reply:
Good spot, fixed.